app/web_framework/cluster_master_application.py

```diff
@@ -296,7 +275,8 @@ class _SlavesHandler(_ClusterMasterBaseAPIHandler):
     def post(self):
         slave_url = self.decoded_body.get('slave')
         num_executors = int(self.decoded_body.get('num_executors'))
-        response = self._cluster_master.connect_slave(slave_url, num_executors)
+        session_id = self.decoded_body.get('session_id')
```

Comment:
Just want to make sure that if we were to deploy the master first before the slaves, that it doesn't blow up.